Heading/Heading.js

```diff
@@ -0,0 +1,163 @@
+/**
```

Comment:
Just a reminder to make sure that changes in #125 get populated here or override these `Heading` files.

Reply:
Merged in #125 after it was updated most recently.